### ISSUE TITLE
Add "pitch" vignette

### ISSUE THREAD

**Issue by krlmlr** [ropensci/drake#231]
- use slides as internal article for pkgdown, https://github.com/krlmlr/drake-pitch
- add words and links to further reading, use this as "Get started" vignette
- think about adapting the *cooking* slides, https://github.com/krlmlr/cooking

**Comment by wlandau** [comment]
Yes, this should definitely replace the [`drake.Rmd` vignette](https://github.com/ropensci/drake/blob/master/vignettes/drake.Rmd) and the [Get Started page](https://ropensci.github.io/drake/articles/drake.html). Right now, it just copies the basic example.

**Comment by wlandau** [comment]
@krlmlr could I delegate this one to you?

**Comment by wlandau** [comment]
I thought this might be nice to have in 5.1.0, but we can absolutely postpone it. Thoughts?

**Comment by krlmlr** [comment]
I can take a stab next week, but not much earlier. Happy to postpone.

**Comment by wlandau** [comment]
Sure, no urgency here. Removing from the 5.1.0 milestone.

**Comment by lorenzwalthert** [comment]
Just stumbled upon that. I feel like the basic example vignette is already quite complicated and I think it would be beneficial to have a more basic example to get people started. From looking at the basic example for about 10 minutes, I did not understand the core of drake, because IMHO it tries to do too much. I only understand the basic mechanisms after looking at the slides linked above by @krlmlr.

**Comment by lorenzwalthert** [comment]
ok, cool. Thanks @wlandau, I appreciate your work on drake.

**Comment by wlandau** [comment]
The documentation is much improved now. There is a new `drake_example("main")` based on [Kirill's slides](https://krlmlr.github.io/drake-pitch), and it is front and center in the README, the [`drake.Rmd` vignette](https://github.com/ropensci/drake/blob/master/vignettes/drake.Rmd), and the [Get Started page](https://ropensci.github.io/drake/articles/drake.html). The old "basic example" is renamed the "mtcars example". I will make an announcement early this week. Thanks again for the nudge.

**Comment by lorenzwalthert** [comment]
That's awesome.

**Comment by krlmlr** [comment]
Thanks for taking care of it! I like the simple language, and the brevity of the presentation. Just very minor remarks:

- "`make()` to actually make" sounds a bit odd, I sometimes frame this like "...to perform the computation" but maybe there's a shorter and more fitting term.
- "You will make mistakes" sounds pessimistic, maybe "After a while you'll notice potential for improvement" or even "Don't worry about mistakes, you can always go back"

**Comment by krlmlr** [comment]
Also, I forgot to credit Jenny's slide in my slide deck, could you please add a link to https://speakerdeck.com/jennybc/zen-and-the-art-of-workflow-maintenance?slide=55?

CC @jennybc.

**Comment by wlandau** [comment]
Gosh, those comments make such a difference! See 617500fc7a13ebe6f89c752c9ab69639ffa95e9b. I tweaked the language and cited Jenny's presentation instead of yours.
